src/lib/viewers/doc/DocLoader.js

```diff
@@ -5,9 +5,81 @@ import PresentationViewer from './PresentationViewer';
 import RepStatus from '../../RepStatus';
 import { ORIGINAL_REP_NAME, STATUS_SUCCESS } from '../../constants';
 
-// Order of the viewers matters. Prefer original before others. Go from specific to general.
```

Comment:
Can we make a note to move out document formats to a different file, and make a format constants file? Maybe `formats.js`, or something like that? Thoughts?